ESMF/GOCART2G_GridComp/CA2G_GridComp/CCMI_REF-C2/CA2G_instance_CA.br.rc

```diff
@@ -26,7 +21,7 @@ hydrophobic_fraction: 0.5
 fscav: 0.0  0.4
 
 # particle radius
-particle_radius_microns: 0.0212  0.0212
+particle_radius_microns: 0.35 0.35
```

Comment:
Some of the shifts in parameter values are disconcerting.   This one in particular which is > 10x.